libflux/flux/src/wasm/mod.rs

```diff
@@ -35,3 +67,34 @@ pub fn wasm_find_var_type(source: &str, file_name: &str, var_name: &str) -> JsVa
+        if flux_identifier == i.path {
+            // return that doc package
+            doc = JsValue::from_serde(&serde_json::to_string(&i.values).unwrap()).unwrap();
+        }
+    }
+
+    doc
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use wasm_bindgen_test::*;
+
+    #[wasm_bindgen_test]
+    pub fn json_docs_test() {
+        let csv_doc = r#"[{"pkgpath":"csv","name":"from","doc":"<p>from is a function that retrieves data from a comma separated value (CSV) data source.</p>\n<p>A stream of tables are returned, each unique series contained within its own table.\nEach record in the table represents a single point in the series.</p>\n<h2>Parameters</h2>\n<ul>\n<li>\n<p><code>csv</code> is CSV data.</p>\n<p>Supports anonotated CSV or raw CSV. Use mode to specify the parsing mode.</p>\n</li>\n<li>\n<p><code>file</code> if the file path of the CSV file to query.</p>\n<p>The path can be absolute or relative. If relative, it is relative to the working\ndirectory of the <code>fluxd</code> process. The CSV file must exist in the same file\nsystem running the <code>fluxd</code> process.</p>\n</li>\n<li>\n<p><code>mode</code> is the CSV parsing mode. Default is annotations.</p>\n<p>Available annotation modes:\nannotations: Use CSV notations to determine column data types.\nraw: Parse all columns as strings and use the first row as the header row\nand all subsequent rows as data.</p>\n</li>\n</ul>\n<h2>Query anotated CSV data from file</h2>\n<pre><code>import &quot;csv&quot;\n\ncsv.from(file: &quot;path/to/data-file.csv&quot;)\n</code></pre>\n<h2>Query raw data from CSV file</h2>\n<pre><code>import &quot;csv&quot;\n\ncsv.from(\n  file: &quot;/path/to/data-file.csv&quot;,\n  mode: &quot;raw&quot;\n)\n</code></pre>\n<h2>Query an annotated CSV string</h2>\n<pre><code>import &quot;csv&quot;\n\ncsvData = &quot;\n#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,double\n#group,false,false,false,false,false,false,false,false\n#default,,,,,,,,\n,result,table,_start,_stop,_time,region,host,_value\n,mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,east,A,15.43\n,mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,east,B,59.25\n,mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,east,C,52.62\n&quot;\n\ncsv.from(csv: csvData)\n\n</code></pre>\n<h2>Query a raw CSV string</h2>\n<pre><code>import &quot;csv&quot;\n\ncsvData = &quot;\n_start,_stop,_time,region,host,_value\n2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,east,A,15.43\n2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,east,B,59.25\n2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,east,C,52.62\n&quot;\n\ncsv.from(\n  csv: csvData,\n  mode: &quot;raw&quot;\n)\n</code></pre>\n","typ":"(?bucket:string, ?bucketID:string, ?host:string, ?org:string, ?orgID:string, ?token:string) => [{A with _value:B, _time:time, _measurement:string, _field:string}]"}]"#;
```

Comment:
I think you meant in this part of the text:
`<code>file</code> if the file path of the CSV file to query.`
to be 
`<code>file</code> **is** the file path of the CSV file to query.`

Reply:
This is a problem with our hand written documentation of the csv.flux file not with this issue. Let me merge this through to avoid future problems with CI and rebasing, and then I will put up a patch to fix the csv.flux file and this test.

Reply:
This is a problem with our hand written documentation of the csv.flux file not with this issue. Let me merge this through to avoid future problems with CI and rebasing, and then I will put up a patch to fix the csv.flux file and this test.

Reply:
![Screen Shot 2021-07-19 at 11 18 10 AM](https://user-images.githubusercontent.com/53279816/126215710-7510bfa9-44b0-46d0-aacf-c78532b085c3.png)